src/decoder/lattice-faster-decoder.cc

```diff
@@ -715,15 +718,13 @@ BaseFloat LatticeFasterDecoder::ProcessEmitting(DecodableInterface *decodable) {
     StateId state = best_elem->key;
     Token *tok = best_elem->val;
     cost_offset = - tok->tot_cost;
-    for (fst::ArcIterator<fst::Fst<Arc> > aiter(fst_, state);
+    for (fst::ArcIterator<FstType> aiter(fst, state);
          !aiter.Done();
          aiter.Next()) {
-      Arc arc = aiter.Value();
+      const Arc arc = aiter.Value();
```

Comment:
The point here is not the const-- that doesn't change anything.  The point is to make it a reference, e.g.
`const Arc &arc`.  That avoids a copy.  There is no need to make it const if it's not a reference.